packages/es-components/src/components/containers/tabPanels/TabPanel.specs.js

```diff
@@ -1,11 +1,18 @@
 /* eslint-env jest */
 
 import React from 'react';
-import { cleanup } from 'react-testing-library';
+import { cleanup, wait } from 'react-testing-library';
 
 import TabPanel from './TabPanel';
 import { renderWithTheme } from '../../util/test-utils';
 
+/*
+  The addition of async support has introduced a strange log message in the tests...
+    "When testing, code that causes React state updates should be wrapped into act(...):"
+  I was unable to resolve this, but this is where my research led me...
+    https://github.com/testing-library/react-testing-library/issues/535#issuecomment-576816390
+*/
```

Comment:
I believe we should be able to resolve this by wrapping things like the `click` in line 38 in an `act` call. https://reactjs.org/docs/test-utils.html#act

Reply:
I've tried different variations of it and none of them seem to work..
- `import { cleanup, wait, act } from 'react-testing-library';`
- import { act } from "react-dom/test-utils"
- `act(() => { ... });`
- `await act(async () => { ... });`

Reply:
Well, I'll be...

The README addresses this exact issue...
> There is a known compatibility issue with React DOM 16.8 where you will see the
> following warning:
> 
> ```
> Warning: An update to ComponentName inside a test was not wrapped in act(...).
> ```
> 
> If you cannot upgrade to React DOM 16.9, you may suppress the warnings by adding
> the following snippet to your test configuration
> ([learn more](https://github.com/testing-library/react-testing-library/issues/281)):
> 
> ```js
> // this is just a little hack to silence a warning that we'll get until we
> // upgrade to 16.9: https://github.com/facebook/react/pull/14853
> const originalError = console.error
> beforeAll(() => {
>   console.error = (...args) => {
>     if (/Warning.*not wrapped in act/.test(args[0])) {
>       return
>     }
>     originalError.call(console, ...args)
>   }
> })
> 
> afterAll(() => {
>   console.error = originalError
> })
> ```
https://github.com/testing-library/react-testing-library/blob/master/README.md#suppressing-unnecessary-warnings-on-react-dom-168

This looks like its working. Pushing shortly...